src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -493,7 +509,9 @@ impl<'a> Linker for GccLinker<'a> {
             if !self.is_ld {
                 arg.push("-Wl,")
             }
-            arg.push("--version-script=");
+            if !is_windows {
+                arg.push("--version-script=")
+            }
         }
 
         arg.push(&path);
```

Comment:
To clarify, `gcc` and `ld` really accept `foo.def` files on the command line without any prefix? (Even `link.exe` wants a prefix `/DEF:foo.def`.)

Reply:
Yeah, it surprised me but both LD and LLD work that way. Only `.def` extension seems importatnt.

Reply:
Could you add a comment reassuring that passing `foo.def` is indeed what we need and we didn't forget the prefix.